src/modules/peek/Peek.FilePreviewer/Previewers/UnsupportedFilePreviewer/UnsupportedFilePreviewer.cs

```diff
@@ -155,16 +148,21 @@ private bool HasFailedLoadingPreview()
         }
 
         // TODO: Move this to a helper file (ImagePrevier uses the same code)
```

Comment:
ImagePrevier => ImagePreviewer. Sorry I know this is not your change, just thought I'd point it out :D

Reply:
Hmm, I don't see it anymore, I think I reverted this?